pkg/anago/release.go

```diff
@@ -419,4 +426,20 @@ func (d *DefaultRelease) CreateAnnouncement() error {
 	return nil
 }
 
-func (d *DefaultRelease) Archive() error { return nil }
+// Archive stores the release artifact in a bucket along with
+// its logs for long term conservation
+func (d *DefaultRelease) Archive() error {
+	// Create a new options set for the release archiver
+	archiverOptions := &release.ArchiverOptions{
+		ReleaseBuildDir: filepath.Join(workspaceDir, d.options.BuildVersion),
+		LogsDirectory:   filepath.Join(workspaceDir, "tmp"),
+		StageGCSPath:    "gs://" + filepath.Join(d.options.Bucket(), "stage"),
```

Comment:
let's use `object.GcsPrefix` for this.

Reply:
These options are gone, but internally i'm using the constant